spec/models/searchgov_document_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe SearchgovDocument do
+  let(:valid_attributes) do
+    {
+      web_document: 'Document body',
+      headers: { content_type: 'text/plain',
+                 etag: '123' },
+      searchgov_url_id: 1
+    }
+  end
+  let(:doc) { described_class.new(valid_attributes) }
+
+  describe 'schema' do
+    it { is_expected.to have_db_column(:web_document).of_type(:text) }
+    it { is_expected.to have_db_column(:headers).of_type(:json) }
+    it { is_expected.to have_db_column(:tika_version).of_type(:decimal) }
```

Comment:
We should probably test all the column options:
```ruby
    it { is_expected.to have_db_column(:headers).of_type(:json).with_options(null: false) }

    it do
      is_expected.to have_db_column(:web_document).
        of_type(:text).with_options(null: false, limit: 4_294_967_295)
    end

    it do
      is_expected.to have_db_column(:tika_version).
        of_type(:decimal).with_options(precision: 10, scale: 4, default: nil)
    end
```    
 
 I used to think this was testing overkill until I learned the hard way that migrations silently ignore options that are set incorrectly...